tests/fuzz/cli_received.cpp

```diff
@@ -53,6 +52,21 @@ static int CliOutput(void *aContext, const char *aFormat, va_list aArguments)
     return vsnprintf(nullptr, 0, aFormat, aArguments);
 }
 
+void AdvanceTime(otInstance *aInstance, uint32_t aDuration)
+{
+    uint32_t time = otPlatAlarmMilliGetNow() + aDuration;
+
+    while (otPlatAlarmMilliGetNow() <= time)
```

Comment:
It probably does not matter in the way we use the `AdvantceTime()` here, but may be safer to do something like this (which should work even under overflow situations?

```
{
    uint32_t start = otPlatAlaramMilliGetNow();

    while (otPlatAralmMilliGetNow() - start <= aDuration)
    {
```
BTW, realized it may be better to do the same in unit tests (will add it in upcoming PR).